bin/pygrb/pycbc_pygrb_plot_coh_ifosnr

```diff
@@ -33,7 +33,9 @@ from matplotlib import pyplot as plt
 from matplotlib import rc
 import numpy
 import scipy
+
 import pycbc.version
+from pycbc.detector import Detector
 from pycbc import init_logging
 from pycbc.detector import Detector
```

Comment:
Duplicate import of pycbc.detector.Detector

Reply:
Thanks - again this would be solved once the above mentioned issue is solved